packages/react-ui-core/src/Dropdown/DropdownMenu.js

```diff
@@ -46,30 +50,36 @@ export default class DropdownMenu extends Component {
     document.addEventListener('keydown', this.onKeyDown)
```

Comment:
should this be in componentDidMount()?

Reply:
This was there before, but it looks like it yeah. Where do you think it should go?

Reply:
I've never used it in constructor only after the component was mounted in componentDidMount() - there may be a valid reason to do so so though that's why I was asking.

Reply:
I think it is better suited for `componentDidMount` as well.

Reply:
Not sure who put it in `constructor` (decent chance it was me 😉 ) My bad if so

Reply:
Am I missing something? I haven't changed this and it's in `componentDidMount`. Did you expand the code?

Reply:
Nope, you aren't missing anything... it is in `componentDidMount`. Should have double checked. What are you talking about @grenti ??

Reply:
Sorry about that - I thought I saw it in the constructor. LGTM 👍